generator/ServiceClientGeneratorLib/Generators/BaseGenerator.cs

```diff
@@ -526,19 +526,40 @@ public void FormatPropertyDocumentation(Member member, string documentationPrefa
         {
             WriteCommentBlock("        ", evenPublisherDocumentation);
         }
+        if (isCollectionProperty)
+        {
+
+        
+        #line default
+        #line hidden
+        
+        #line 164 "C:\codebase\v4\aws-sdk-net-v4\generator\ServiceClientGeneratorLib\Generators\BaseGenerator.tt"
+this.Write("        /// <para />\r\n");
+
+        
+        #line default
+        #line hidden
+        
+        #line 166 "C:\codebase\v4\aws-sdk-net-v4\generator\ServiceClientGeneratorLib\Generators\BaseGenerator.tt"
+
+            WriteCommentBlock("        ", "Starting with version 4 of the SDK this property will default to null and if no data for this property is returned");
```

Comment:
nit: I would make this two sentences. it took me a bit to understand.

"Starting with version 4 of the SDK this property will default to null. If the service returns no data for this property it will be null."

Reply:
Done